cypress/support/commands.ts

```diff
@@ -199,7 +209,11 @@ Cypress.Commands.add('setupOwner', (payload) => {
 });
 
 Cypress.Commands.add('enableFeature', (feature) => {
-	cy.task('enable-feature', feature);
+	cy.task('enable-feature', { feature, enabled: true });
+});
+
+Cypress.Commands.add('disableFeature', (feature) => {
+	cy.task('enable-feature', { feature, enabled: false });
```

Comment:
maybe this needs to be renamed then

Reply:
How come? The task itself reflects naming of the BE endpoint and it'll be used only for these two commands. The tests would always use just `disableFeature` or `enableFeature`

Reply:
because `enable-feature` should also be able to disable a feature.. I would rename this and the endpoint.. a restful approach here would be `PATCH /feature` and `{enabled: true/false}` as body

Reply:
Fair point. I've updated it. Can you have one more look?